packages/nextjs/components/Header.tsx

```diff
@@ -101,7 +119,71 @@ export const Header = () => {
           <HeaderMenuLinks />
         </ul>
       </div>
+
       <div className="navbar-end flex-grow mr-4">
+        <div className="flex">
+          {address ? (
+            isInAllowList ? (
+              isCheckedIn ? (
+                <div
+                  className="tooltip tooltip-bottom mr-5"
+                  data-tip="You are in the allowlist and you have checked in"
+                >
+                  <button className="btn-active bg-transparent text-2xl">
+                    <svg
+                      xmlns="http://www.w3.org/2000/svg"
+                      fill="none"
+                      viewBox="0 0 24 24"
+                      strokeWidth={1.5}
+                      stroke="currentColor"
+                      className="size-6"
+                    >
+                      <path strokeLinecap="round" strokeLinejoin="round" d="m4.5 12.75 6 6 9-13.5" />
+                    </svg>
+                  </button>
```

Comment:
Ohh I think you directly copied the svg from HeroIcons website, we actually have [`@heroicons/react`](https://github.com/tailwindlabs/heroicons?tab=readme-ov-file#react) installed in Scaffold-ETH 2 

So you import them like components as we have done of BugIcon etc example : 

```tsx
import { Bars3Icon, BugAntIcon } from "@heroicons/react/24/outline";
```